### DISCUSSION TITLE
[Unstable v0.20.5.0] Feedback on Scrapper Mechanic tree

### DISCUSSION THREAD

**Discussion by AhpiTai** [FakeFishGames/Barotrauma#10349] [Unstable feedback]
I've seen other discussions on Mechanic's talent rework but I wanted to write an isolated post about the Tier 4 talents, 'Mass production' and 'Residual waste'. First of all, the new talents are great and exciting overall. Also, I understand that I don't know the developers' philosophy for their decisions in this rework. This is just from a mechanic's perspective

 'Mass production and 'Residual waste' will be referred to as "MP" and "RW" for the remainder of the post.

 It's much better than in this unstable version that MP no longer has to compete with 'Tool maintenance', but MP still seems rather weak. The idea of MP and RW is really nice but a 1/10 chance doesn't seem consistent enough. It's almost like you don't have the talent at all. The intent behind these talents would feel much better if;

1. Chance to trigger was increased slightly (e.g from  '1 in 10' to at least a '1 in 8' or '1 in 5' chance)
2. Or the trigger for the talents wasn't RNG at all.

 Being in charge of resource management is the one thing a Mechanic does well and makes it distinct from other classes aside from just being _the_ repair man. The entire crew is usually already equipped with the tools they need anyway and often repair things before you're needed.

The talents, of MP and RW don't seem consistent enough for it to be necessary to have your Scrapper Mechanic in charge of resource management, even though that is the tree's forte. Let alone even having a satisfying impact in gameplay.
 
I hope this is at least considered.

**Comment by Udrakan** [comment]
![image](https://user-images.githubusercontent.com/59519434/201496885-f2cd78a6-dbf1-4bd4-9d7d-0140ec295787.png)
Mass Production could use something else entirely. A suggestion on discord was made that would help mechanic become much better in charge of production:

**Mass Production:
-Increases fabrication speed as long as you keep crafting the same item up to x %**

-alternatively:

**Mass Production:
-Increases fabrication speed by up to x% and a chance to not consume 1st resource in a recipe by y% as long as you keep crafting the same item**

RW would need a boost for it to be on par with such a nice perk though
1 suggestion that comes to mind:

**Residual waste
-Deconstructing items has a z% chance to additionally yield scrap**

This would tie in nicely with the scrap cannon at the bottom of the tree HOWEVER (as pointed out by Uber) this could lead to duping through recursive crafting with the Scrap Savant perk (double the yield from scrap) so perhaps another suggestion would be better.


Btw this chance not to consume resources when crafting perks and talents are boring and annoying in other games as well. If they are implemented at all, they should come with audiovisual clue when they trigger, but I'd rather see them as a secondary to another talent or upgrade.
(Cyberpunk 2077 perk)
![image](https://user-images.githubusercontent.com/59519434/201496966-fad0651d-0fc5-4c99-a650-de7e66b6d5bd.png)

**Reply by AhpiTai** [reply]
I agree with your opinion and the ideas provided. As is now, it seems hard to balance. It's either inconsistent and sucks or just leads to item duping and stockpiling. Doesn't help it's RNG for something you'll be using all the time. Unlike finding materials in wrecks which is not a consistent occurrence.

**Comment by cshingle** [comment]
I agree that this will likely lead to boring item duping "exploits". I think it would be better to have a chance for an item to complete instantly and at a higher tear.
